packages/components/src/font-size-picker/types.ts

```diff
@@ -1,9 +1,26 @@
+export type FontSizePickerMode = 'predefined' | 'custom' | 'both';
```

Comment:
I'm wondering if we should avoid calling it "both", in case there will be more mode options added in the future. Maybe something like "default", "full", or "all"? I guess "default" will be the most future-proof since we don't know if the future default will include all possible modes.